### PR TITLE
ME: edit record distributions

### PR DESCRIPTION
**Pull request by LHBruneton-C2C**
### Description

This PR introduces the section on associated resources, allowing to add both download and service as distributions.

TODO: split and share components between annexes and associated resources. 

### Screenshots

image
image

### Quality Assurance Checklist

- [X] Commit history is devoid of any _merge commits_ and readable to facilitate reviews
- [ ] If **new logic** ⚙️ is introduced: unit tests were added
- [X] If **new user stories** 🤏 are introduced: E2E tests were added
- [ ] If **new UI components** 🕹️ are introduced: corresponding stories in Storybook were created
- [ ] If **breaking changes** 🪚 are introduced: add the `breaking change` label
- [ ] If **bugs** 🐞 are fixed: add the `backport <release branch>` label
- [ ] The [documentation website](docs) 📚 has received the love it deserves

### CODE REVIEW

File: libs/api/metadata-converter/src/lib/iso19139/read-parts.ts [hunk 1]
```diff
@@ -381,6 +381,7 @@ export function extractDatasetOnlineResources(
     mapArray(
       ([isService, isDownload, protocol, url, name, description, mimeType]) => {
         if (isService) {
+          //FIXME: allow writing identifier only for wms and wfs?
```
Comment (LHBruneton-C2C) [inline libs/api/metadata-converter/src/lib/iso19139/read-parts.ts:384]: @jahow This is the only point on which I'm not sure how to fix the converter. As it is currently, for service protocols other than WMS and WFS, the identifier is not lost, but it's transferred to the name property.

This is particularly disturbing when reloading a record after it's been first saved as draft. The attributes aren't kept as they were first inputted.

**Comment by jahow** [comment]
Did a quick test and found this:
* When trying to upload a file as a distribution, this error appears in the console and nothing happens:
image

* When clicking on the main block, the upload file dialog doesn't open

File: libs/ui/inputs/src/lib/url-input/url-input.component.ts [hunk 2]
```diff
@@ -12,16 +19,23 @@ import { Subject } from 'rxjs'
...
   rawChange = new Subject<string>()
   @Output() valueChange = this.rawChange.pipe(filter((v) => !!v))
 
   constructor(private cd: ChangeDetectorRef) {}
 
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.value) {
+      console.log('changes.value', changes.value)
+    }
+  }
```
Comment (jahow) [inline libs/ui/inputs/src/lib/url-input/url-input.component.ts:37]: to delete

**Comment by fvanderbiest** [comment]
Just chiming in - "OGC API" title is misleading because there are so many OGC APIs  (OGC API Maps, Features, Tiles...)
Have a look at https://ogcapi.ogc.org/#standards

**Comment by jahow** [comment]
> Just chiming in - "OGC API" title is misleading because there are so many OGC APIs (OGC API Maps, Features, Tiles...) Have a look at https://ogcapi.ogc.org/#standards

On the other hand it makes sense because an OGC API endpoint can provide many different information at the same place I think

**Pull request merged**